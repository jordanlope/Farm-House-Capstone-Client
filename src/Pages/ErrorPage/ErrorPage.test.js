import React from 'react';
import ReactDOM from 'react-dom';
import ErrorPage from './ErrorPage';
import renderer from 'react-test-renderer';

describe('ErrorPage component', () => {
    // Smoke test
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<ErrorPage errorMessage={'This is a test'}/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    // Snapshot test
    it('renders the UI as expected', () => {
        const tree = renderer
            .create(<ErrorPage errorMessage={'This is a test'}/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
})