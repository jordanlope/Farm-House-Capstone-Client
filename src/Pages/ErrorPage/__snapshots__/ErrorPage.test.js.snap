// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage component renders the UI as expected 1`] = `
<h1>
  Error: 
  This is a test
</h1>
`;
